src/data/config.js

```diff
@@ -11,6 +11,7 @@ import { ACTIONS } from './constants';
  */
 export const DEFAULTS = {
   panorama           : null,
+  addRequestHeaders  : null,
```

Comment:
why it is a function and not an object ?

```js
if (this.config.requestHeaders) {
  this.loader.setRequestHeaders(this.config.requestHeaders);
}
```

Reply:
I need this callback function to set the Authorization header with a valid bearer token just before loader.load is called. So it can't be a object in the config because it's initiate only once when the viewer is created.